pkgs/tools/package-management/reuse/default.nix

```diff
@@ -4,13 +4,13 @@ with python3Packages;
 
```

Comment:
Can you remove the with over everything and only apply it where it is needed?

Reply:
With pleasure.